src/Sdk/Exceptions/ValidationException.php

```diff
@@ -0,0 +1,54 @@
+<?php
```

Comment:
Because all your exceptions define the same `$subCode` property and the same constructor, you should have a base exception instead of duplicating the code.